package.json

```diff
@@ -58,7 +58,7 @@
       }
     },
     "start": {
-      "command": "babel-node bin/server",
+      "command": "npm run lint && babel-node bin/server",
```

Comment:
Im not sure about using it on start, it means a lint issue in development will block the dev from running the app?

Maybe best on the deploy tasks just?

You're right that it's poor that there were existing lint failures, but I think in a running project, we'd expect that to be picked up by CI (rather than enforcing the slower dev process onto everyone).